src/components/MAPI/apps/permissions/usePermissionsForApps.ts

```diff
@@ -65,6 +66,21 @@ export function usePermissionsForApps(
     }
   }, [error]);
 
+  const isLoading =
+    typeof appAccess === 'undefined' &&
+    typeof error === 'undefined' &&
+    isValidating;
+
+  if (isLoading) {
+    computed.canGet = undefined;
```

Comment:
Because the permissions for `apps` are calculated with `SelfSubjectAccessReviews` rather from a permission map, we return `undefined` while the access is being fetched so that we can handle loading states in the UI better.